src/composables/useClient.ts

```diff
@@ -55,43 +57,62 @@ export function useClient() {
   }
 
   async function sendEIP712(space, type, payload) {
```

Comment:
> The only thing I haven't tested is GnosisSafe, will it be enough to create a safe for myself and go through the flow again?

Gnosis safe doesn't use `sendEIP712` so it will not be affected.

Reply:
It's better to disable alias for Safe for now, Safe is managed with multiple signers moving to alias would remove the role of signers and give authority to a single user it's not ideal.

Reply:
> I didn't use aliasing for the usePersonalSign, I assumed Gnosis should always require the signature but correct me if I wrong, still digging into it 🙈

`usePersonalSign` will be deprecated in the very near future, so yeah we don't need to implement aliased voting for it

Reply:
Ok, so if I understand correctly should be fine as is now?

Reply:
Yep, I haven't tested much yet but it looks good